### PR TITLE
chore: release 0.3.8

### PR DESCRIPTION
**Pull request by vigsterkr**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -18,6 +18,15 @@
...
 =======
 
+`0.3.8 <https://github.com/SwissDataScienceCenter/calamus/compare/v0.3.7...v0.3.8>`__ (2021-04-26)
+--------------------------------------------------------------------------------------------------
+
+Fixes
+~~~~~
+
+- Fixed an issue when deserializing raw fields containing JSON-LD.
+  (`PR #58 <https://github.com/SwissDataScienceCenter/calamus/pull/58>`__)
```
Comment (mohammad-sdsc) [inline CHANGES.rst:28]: To me this looks to be more of a feature than a fix; a new RawJsonLD is added. Should we change this to a feature?

**Review by mohammad-sdsc — COMMENTED** [review]
Thanks Viktor! I've made a comment about the changelog.

File: CHANGES.rst (hunk 1 above)
Comment (vigsterkr) [reply CHANGES.rst:28]: sure thing! lemme forcepush the change :)

**Review by mohammad-sdsc — APPROVED** [review]
Thanks!

**Pull request merged**